import-service/serverless.ts

```diff
@@ -0,0 +1,78 @@
+import type {AWS} from '@serverless/typescript';
+import importProductsFile from "./src/functions/import-products-file";
+import importFileParser from "./src/functions/import-file-parser";
+
+const IMPORT_BUCKET_NAME = 'game-store-uploaded';
+
+const serverlessConfiguration: AWS = {
+    service: 'import-service',
+    frameworkVersion: '2',
+    custom: {
+        webpack: {
+            webpackConfig: './webpack.config.js',
+            includeModules: true,
+        },
+    },
+    plugins: ['serverless-webpack'],
+    provider: {
+        name: 'aws',
+        runtime: 'nodejs14.x',
+        region: 'us-east-1',
+        apiGateway: {
+            minimumCompressionSize: 1024,
+            shouldStartNameWithService: true,
+        },
+        environment: {
+            AWS_NODEJS_CONNECTION_REUSE_ENABLED: '1',
+            PG_HOST: 'mydb.cxdn0u6xwgem.us-east-1.rds.amazonaws.com',
```

Comment:
It's not important, but I would say it's better to have db info only in services where you really need it